README.md

```diff
@@ -48,7 +48,11 @@ After you have all the prerequisites installed and have configured your
     make terraform-init
     ```
 
-5. Create the VMs for the validators and Prometheus as specified in `./testnet.toml`   
+41. If you are using `scripts/runtests.py`, execut it now to update your `./testnet.toml` according to your templates. 
+    Use the `-s` flag to run it just once, as in `python3 runtests.py -l log.log -o flood_options.json -s`
+    Otherwise, ensure that your `./testnet.toml` is correct.
```

Comment:
Yes, since the numbering in markdown doesn't really reflect the output numbering, I thought it would be better to use intermediate numbers, as in this example, 4.1, except that it is not really 4.1 but 41 :), instead of touching all the other lines. But I fix it for consistency.